src/Microsoft.ML.Data/DataView/RowToRowMapperTransform.cs

```diff
@@ -367,7 +367,7 @@ public Cursor(IChannelProvider provider, DataViewRowCursor input, RowToRowMapper
                 : base(provider, input)
             {
                 var pred = parent.GetActiveOutputColumns(active);
-                _getters = parent._mapper.CreateGetters(input, pred, out _disposer);
```

Comment:
>_getters = parent._mapper.CreateGetters(input, pred, out _disposer); [](start = 16, length = 68)

lets revert this and use a thread safe dictionary for now. I want to think a little more on this.